src/CMakeLists.txt

```diff
@@ -56,49 +41,9 @@ else()
 	target_compile_options(${PROJECT_NAME} PRIVATE -Wall $<$<CONFIG:DEBUG>:-g3 -Og -Wextra -Wpedantic -pedantic> $<$<CONFIG:RELEASE>:-DNDEBUG -g0 -O3 -mtune=native -march=native>)
```

Comment:
`"$<$<CONFIG:RELEASE>:/O2 /GL>"` and
```
target_link_options(${PROJECT_NAME} PRIVATE "$<$<CONFIG:Release>:/LTCG>")
```
can be skipped for Windows since CMake 3.13 introduces enabling IPO for MSVC via `	set_target_properties(${PROJECT_NAME} PROPERTIES INTERPROCEDURAL_OPTIMIZATION TRUE)
`
As such, all the MSVC Options could be skipped. Although you might want to add `/utf-8` to the Windows compile options. There once was an encoding problem with Asian users.

Also: `-DNDEBUG -O3` can be removed from the other compile options as they are automatically set by CMake for a Release configuration.